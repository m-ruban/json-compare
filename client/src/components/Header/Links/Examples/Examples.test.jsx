import React from 'react';
import { render } from 'enzyme';
import Examples from 'components/Header/Links/Examples';

describe('<Examples />', () => {
    it('Renders correctly', () => {
        const output = render(<Examples />);
        expect(output).toMatchSnapshot();
    });
});
