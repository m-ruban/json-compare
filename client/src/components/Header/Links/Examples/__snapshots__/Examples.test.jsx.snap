// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Examples /> Renders correctly 1`] = `
<li
  class="examples dropdown nav-item"
>
  <a
    aria-expanded="false"
    aria-haspopup="true"
    class="examples__toggle dropdown-toggle nav-link"
    href="#"
  >
    Examples
  </a>
  <div
    aria-hidden="true"
    class="bg-dark examples__dropdown dropdown-menu"
    role="menu"
    tabindex="-1"
  >
    <button
      class="examples__dropdown-item dropdown-item"
      role="menuitem"
      tabindex="0"
      type="button"
    >
      Tool Example
    </button>
    <button
      class="examples__dropdown-item dropdown-item"
      role="menuitem"
      tabindex="0"
      type="button"
    >
      Object Example
    </button>
  </div>
</li>
`;
